pynitrokey/cli/nethsm.py

```diff
@@ -983,9 +1059,12 @@ def get_certificate(ctx, api, key_id):
     with connect(ctx) as nethsm:
         if key_id:
             cert = nethsm.get_key_certificate(key_id)
+            try:
```

Comment:
A user should be able to store arbitrary byte strings for which the .decode() call may not necessarily succeeds.
Using .decode() here limits the byte strings the user can store.

Reply:
If `decode` fails, we print the raw bytes (see the `except` block).  The alternative would be to always print e. g. the hex representation, but I think that would be confusing for most use cases.